### ISSUE TITLE
Not able to make PowerMock work in a junit plugin test

### ISSUE THREAD

**Issue by johanhaleby** [jayway/powermock#402]
_From [vijetha2...@gmail.com](https://code.google.com/u/106470429328666695142/) on April 20, 2012 13:32:55_

What steps will reproduce the problem? 1. Create a fragment plugin (test plugin)for a plugin project on Eclipse.
2. Mock a static class.
3. Run the test as a JUnit Plugin Test What is the expected output? What do you see instead? Expected to mock the static method. 
Instead, "java.lang.Exception: No runnable methods" exception is got What version of the product are you using? On what operating system? powermock-mockito-junit-1.4.12; Windows 7. Please provide any additional information below. The test runs perfectly well with the same lines of code when run as a normal JUnit test.

_Original issue: http://code.google.com/p/powermock/issues/detail?id=382_

**Comment by johanhaleby** [comment]
_From [Kidder1...@googlemail.com](https://code.google.com/u/109865693020220977894/) on August 30, 2012 07:15:17_

Any updates on this bug?  This is a real blocker for me.

**Comment by johanhaleby** [comment]
_From [johan.ha...@gmail.com](https://code.google.com/u/105676376875942041029/) on August 30, 2012 11:05:14_

Sorry, no updates. Please help out by investigating why this happens.

**Comment by katy246-old** [comment]
Is this still an issue?

**Comment by pavanbhosle** [comment]
yes i still see this issue.. plz help if anybody found a solution for this
